apps/i18n/aichat/en_us.json

```diff
@@ -15,11 +15,14 @@
   "chatEventDescriptions_clearChat": "The user cleared the chat workspace.",
   "chatEventDescriptions_loadLevel": "The user loaded the aichat level.",
   "chatEventDescriptions_waitForResponse": "Waiting for response",
-  "chatMessage_showMessage": "Show message",
-  "chatMessage_hideMessage": "Hide message",
+  "chatMessage_showMessage": "Show profane message",
```

Comment:
Will it always be "profane" that a message will be flagged (eg, PII)? Should it just be "show flagged message"?

Reply:
Good point, yeah probably makes sense to be consistent with the "flagged" terminology.